alma/endpoints/exports.py

```diff
@@ -0,0 +1,65 @@
+from functools import partial
+from datetime import datetime
+from io import BytesIO
+
+from . import Base
+from ..entities import Export
+from ..paginated_results import PaginatedResults
+
+
+EXPORT_TYPES = ['payments', 'accounting', 'accounting_for_payout']
+
+
+class ExportsException(Exception):
+    pass
+
+
+class Exports(Base):
+    EXPORTS_PATH = "/v1/data-exports"
+
+    def create(self, export_type: str = None, payout_id: str = None,
+               start: datetime = None, end: datetime = None):
+        """ Create a new export"""
+        if export_type not in EXPORT_TYPES:
+            raise ExportsException("%s is not an availale type" % export_type)
+
+        data = dict(type=export_type)
+        if payout_id:
+            data['payout'] = payout_id
+
+        if start:
+            data['start'] = int(start.timestamp())
+
+        if end:
+            data['end'] = int(end.timestamp())
+
+        response = self.request(self.EXPORTS_PATH).set_body(data).post()
+        return Export(response.json)
+
+    def get_file(self, export_id: str = None, export_format: str = 'csv'):
```

Comment:
```suggestion
    def get_file(self, export_id: str = None, export_format: str = 'csv'):
```

Could we also have an enum type for the `export_format`?
It makes it self-documenting that CSV, XLSX and PDF (on some specific exports) formats are supported